gitlab/types.py

```diff
@@ -45,6 +45,21 @@ def get_for_api(self):
         return ",".join(self._value)
 
 
+class ScopesListAttribute(ListAttribute):
+    def set_from_cli(self, cli_value):
+        if not cli_value.strip():
+            self._value = []
+        else:
+            self._value = [item.strip() for item in cli_value.split(",")]
+
+    def get_for_api(self):
+        # Do not comma-split single value passed as string
+        if isinstance(self._value, str):
+            return [self._value]
+
+        return self._value
+
+
```

Comment:
Since you are inheriting from ListAttribute you don't need to duplicate methods. Just override the one you need to and maybe instead of copying the previous comment, add one (or a docstring) about why it needs to be different from the inherited class. Unless I missed a small difference in `set_from_cli` then please correct me :grin:

Reply:
Hi @nejch,

indeed I lost the changes.  I implemented the original code in the __init__.py (some versions ago) and with the split I oversaw to add the changes in all the new files.

If fixed also the other requested changes.

Regards, Kiwi